lighthouse-core/lib/i18n/i18n.js

```diff
@@ -54,6 +54,8 @@ const UIStrings = {
   displayValueByteSavings: 'Potential savings of {wastedBytes, number, bytes}\xa0KiB',
   /** Label shown per-audit to show how many milliseconds faster the page load could be if the user implemented the suggestions. The `{wastedMs}` placeholder will be replaced with the time duration, shown in milliseconds (e.g. 140 ms) */
   displayValueMsSavings: 'Potential savings of {wastedMs, number, milliseconds}\xa0ms',
+  /** Label shown per-audit to show how many HTML elements did not pass the audit. The `{# elements found}` placeholder will be replaced with the number of failing HTML elements. */
+  displayNodes: `{nodeCount, plural, =1 {1 element found} other {# elements found}}`,
```

Comment:
```suggestion
  displayValueElementsFound: `{nodeCount, plural, =1 {1 element found} other {# elements found}}`,
```

(you'll have to make a few other changes for this)